src/api.ts

```diff
@@ -67,6 +67,8 @@ export interface SpawnTokenRevokeResult {
   status: 'ok',
 }
 
+export interface TerminateResult {}
```

Comment:
Looks like the result is `{ status: 'ok' }`: https://github.com/drifting-in-space/p2/blob/main/jamsocket/src/platform/server/api/backend.rs#L166

Reply:
Indeed, good catch.